### PR TITLE
Fixes deadchat

### LINKED ISSUE #24925: Every person that talks in deadchat has like 7 (F) links next to their name
See title

aaaaa

### PR DESCRIPTION
**Pull request by silicons**
Fixes deadchat broadcast code recursively adding follow links for each ghost broadcasted to.
@coiax fix your shit
:cl:
bugfix: Deadchat no longer has huge amount of F's.
/:cl:
Fixes #24925

### CODE REVIEW

**Comment by Cheridan** [comment]
F

**Comment by kevinz000** [comment]
F

**Comment by PKPenguin321** [comment]
(F)

**Comment by PKPenguin321** [comment]
(F)ixes https://github.com/tgstation/tgstation/issues/24925

**Comment by Cyberboss** [comment]
F

**Comment by FantasticFwoosh** [comment]
F

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -401,8 +401,7 @@ Proc for attack log creation, because really why not
 					F = FOLLOW_OR_TURF_LINK(M, follow_target, turf_target)
 				else
 					F = FOLLOW_LINK(M, follow_target)
-				message = "[F] [message]"
+				M << "[F] [message]"
 			else if(turf_target)
 				var/turf_link = TURF_LINK(M, turf_target)
-				message = "[turf_link] [message]"
-		M << "[message]"
+				M << "[turf_link] [message]"
```
Comment (coiax) [inline code/__HELPERS/mobs.dm]: You've removed the ability for non-observers to hear deadchat.

**Comment by coiax** [comment]
Once I accidentally removed the names from all ghosts. So it was just a bunch of anonymous ghosts bitching at each other.

**Comment by XDTM** [comment]
I guess the link has to go directly to the observers, or it will convert into a string.

File: code/__HELPERS/mobs.dm [hunk 2]
```diff
@@ -401,8 +401,7 @@ Proc for attack log creation, because really why not
 					F = FOLLOW_OR_TURF_LINK(M, follow_target, turf_target)
 				else
 					F = FOLLOW_LINK(M, follow_target)
-				message = "[F] [message]"
+				M << "[F] [message]"
 			else if(turf_target)
 				var/turf_link = TURF_LINK(M, turf_target)
-				message = "[turf_link] [message]"
-		M << "[message]"
```
Comment (AnturK) [inline code/__HELPERS/mobs.dm]: Switch this to to_chat, and just add temporary variable instead of sending it directly.

**Comment by optimumtact** [comment]
TO chat nigga

**Pull request merged**